kolibri/utils/options.py

```diff
@@ -361,6 +361,11 @@ def url_prefix(value):
             "default": "",
             "description": "Additional directories in which Kolibri will look for content files and content database files.",
         },
+        "AUTOMATIC_PROVISION_FILE": {
+            "type": "path",
+            "default": "automatic_provision.json",
```

Comment:
It's a little confusing that this is the default, but the file doesn't actually exist. Two options I can think of:

1. make the default `null`
2. create an actual `automatic_provision.json` in the codebase, and use that to be the "source of truth" for default. This would have the advantage that the it would serve as a self-documenting reference, which would be more likely to stay up-to-date than the "Expects a JSON file with the following format (example values supplied)" comment.

Reply:
The main reason for not having a file is that the default behaviour is *not* to automatically provision.

I think setting the default to empty string would be most explicit, and then forcing people leveraging this functionality to explicitly name the file they want to import from.